src/Uno.UI/UI/Xaml/Media/Imaging/WriteableBitmap.skia.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using Windows.Foundation;
+using Windows.Storage.Streams;
+using Uno.Foundation;
+using Windows.UI.Composition;
+
+namespace Windows.UI.Xaml.Media.Imaging
+{
+	partial class WriteableBitmap
+	{
+		private SkiaCompositionSurface _surface;
+
+		private protected override bool TryOpenSourceSync(int? targetWidth, int? targetHeight, out ImageData image)
+		{
+			var handle = GCHandle.Alloc(_buffer.Data, GCHandleType.Pinned);
```

Comment:
Codacy found an issue: [Remove the unused local variable 'handle'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6199330)